internal/DateTime/tests/DateTimeDecorator-specs.js

```diff
@@ -1,31 +1,34 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import {DateTimeDecorator} from '../';
 
 describe('DateTimeDecorator', () => {
-	test(
-		'should accept an updated JavaScript Date for its value prop',
-		() => {
-			const Picker = DateTimeDecorator({}, function PickerBase () {
-				return <div />;
-			});
-
-			const subject = mount(
-				<Picker
-					title="Date"
-					value={new Date(2000, 0, 1, 12, 30)}
-					locale="en-US"
-				/>
-			);
+	test('should accept an updated JavaScript Date for its value prop', () => {
+		const Picker = DateTimeDecorator({}, function PickerBase ({locale, title, value}) {
+			const minuteValue = value.getMinutes();
+			return <div locale={locale} title={title} >{minuteValue}</div>;
```

Comment:
The prop is available for reading only if it displayed in the DOM. That's why the view is modified